noxfile.py

```diff
@@ -19,26 +19,36 @@ def unit(session, sqlalchemy):
 @nox.session(python=["3.8", "3.9", "3.10", "3.11"])
```

Comment:
**question (non-blocking)**: Is it too early for 3.12 support?

Reply:
definitely not (although it wasn't out when I started this branch  😅 )